### ISSUE TITLE
IPluggableAuthenticationModules

### ISSUE THREAD

**Issue by rezuma** [micheloosterhof/kippo#35]
I was having problems with the import of IPluggableAuthenticationModules. 

It looks like with the new version of Twisted this 

from twisted.cred.credentials import IUsernamePassword, ISSHPrivateKey, \
    IPluggableAuthenticationModules

is not longer supported or valid

**Comment by micheloosterhof** [comment]
Correct. It seems it was removed here: http://twistedmatrix.com/trac/changeset/44862/trunk/twisted/cred/credentials.py
